.github/workflows/external.yml

```diff
@@ -0,0 +1,109 @@
+      - 'develop/**'
+    paths-ignore:
+      - "docs/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  CTEST_OUTPUT_ON_FAILURE: 1
+  CCACHE_DIR: ${{ github.workspace }}/ccache
+  CCACHE_MAXSIZE: 500M
+  CCACHE_KEY_SUFFIX: r2
+
+jobs:
+  eic-shell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+          path: acts
+
+      - uses: actions/checkout@v4
+        with:
+          repository: eic/EICrecon
+          path: EICrecon
+
+      - name: Setup CMVFS
+        uses: cvmfs-contrib/github-action-cvmfs@v5
+
+      - name: Restore ccache
+        id: ccache-restore
+        uses: actions/cache/restore@v4
+        with:
+          path: ${{ env.CCACHE_DIR }}
+          key: ccache-${{ runner.os  }}-${{ github.job }}-${{ env.CCACHE_KEY_SUFFIX }}-${{ github.sha }}
+          restore-keys: |
+            ccache-${{ runner.os  }}-${{ github.job }}-${{ env.CCACHE_KEY_SUFFIX }}-
+
+      - name: Build
+        id: build
+        uses: eic/run-cvmfs-osg-eic-shell@main
+        continue-on-error: true
+        with:
+          platform-release: "eic_xl:nightly"
+          run: |
+            pushd acts
+            echo 99.9.9 > version_number
+            ccache -z && \
+            export CMAKE_INSTALL_PREFIX=$HOME/prefix
+            export CMAKE_CXX_COMPILER_LAUNCHER=ccache
+            cmake -B build -S . \
+              -GNinja \
+              -DACTS_BUILD_EXAMPLES=ON \
+              -DACTS_BUILD_FATRAS=ON \
+              -DACTS_BUILD_PLUGIN_DD4HEP=ON
+            cmake --build build
+            cmake --install build
+            popd
+
+            pushd EICrecon
+            export LD_LIBRARY_PATH="$CMAKE_INSTALL_PREFIX/lib:$LD_LIBRARY_PATH"
+            ccache -z && \
+            env CXXFLAGS="-I$CMAKE_INSTALL_PREFIX/include" \
+                LDFLAGS="-L$CMAKE_INSTALL_PREFIX/lib" \
+            cmake -B build -S . \
+              -GNinja \
+              -DActs_ROOT=$CMAKE_INSTALL_PREFIX
+            cmake --build build
+            popd
+
+            ccache -s
+
+      - name: Find Comment
+        if: ${{ steps.build.outcome == 'failure' && github.event_name == 'pull_request' }}
+        uses: peter-evans/find-comment@v3
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Downstream build failure
+
+      - name: Create or update comment
+        if: ${{ steps.build.outcome == 'failure' && github.event_name == 'pull_request' }}
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Downstream build failure
+
+             - eic-shell (cc @eic/epic-acts-devs)
+          edit-mode: replace
```

Comment:
Seems like this won't get triggered in the PR context, and I think PR jobs also won't have permission to post. 

But what we can do instead is have a workflow trigger when this build job completes using the [`workflow_run`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows?versionId=free-pro-team%40latest&productId=actions&restPage=writing-workflows%2Cchoosing-when-your-workflow-runs%2Cevents-that-trigger-workflows#workflow_run) trigger. 

That requires a few extra steps like saving the PR number as an artifact and checking the other workflows output.

Reply:
Right. I could modify report.yml to download this too and concatenate it to the report. There is an option go with `pull_request_target` too, if we don't expect the action to have changes done to it often.